manager/src/main/java/com/redhat/service/smartevents/manager/workers/quartz/QuartzWorkManagerImpl.java

```diff
@@ -0,0 +1,130 @@
+package com.redhat.service.smartevents.manager.workers.quartz;
+
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.quartz.JobBuilder;
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.SchedulerException;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.quartz.TriggerKey;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.manager.models.ManagedResource;
+import com.redhat.service.smartevents.manager.models.Work;
+import com.redhat.service.smartevents.manager.workers.WorkManager;
+
+import io.quarkus.runtime.Quarkus;
+
+import static com.redhat.service.smartevents.manager.workers.quartz.QuartzWorkConvertor.convertToJobData;
+
+@ApplicationScoped
+public class QuartzWorkManagerImpl implements WorkManager {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QuartzWorkManagerImpl.class);
+
+    // Whilst Quartz supports use of Serialisable objects as JobData, Quarkus configures Quartz to force
+    // use of String keys and values when using a JDBC JobStore. Therefore, these properties are always
+    // stored as Strings and the serialization/de-serialisation handled by RHOSE.
+    // See https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/Quartz.3A.20JDBC.20JobStore.3A.20useProperties
+    static final String STATE_FIELD_ID = "id";
+    static final String STATE_FIELD_ATTEMPTS = "attempts";
+    static final String STATE_FIELD_SUBMITTED_AT = "submittedAt";
+    static final String STATE_FIELD_TYPE = "type";
+
+    @ConfigProperty(name = "event-bridge.resources.workers.job-retry-interval-seconds")
+    long jobScheduleInterval;
+
+    @Inject
+    org.quartz.Scheduler quartz;
+
+    private JobDetail workJob;
+
+    @PostConstruct
+    public void init() {
+        try {
+            workJob = JobBuilder.newJob(WorkJob.class).storeDurably().build();
+            quartz.addJob(workJob, false);
```

Comment:
No. We define one `Job` and then multiple `Trigger`'s to execute the `Job` with different parameters (defined in `Work` i.e. `ManagedResource.id`, submitted date, number of attempts etc). Since we re-use the same `Job` we simply declare it to Quartz once in start-up.